### PR TITLE
Add scripts for dataset download and augmentation

### PR DESCRIPTION
**Pull request by gkielian**
These allow replication of tinystories augmentation (in this case translation) into other languages.

These also unblock the possibility of augmenting with the summaries pre-provided it turns out in the json formatted stories (these actually are only in the json and not in the text stories which is a fortuitous find).

### CODE REVIEW

File: data_augmentation/tinystories_en.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    if [ ! -f "${output}" ]; then
+
+      if [ "$processes" -ge 20 ]; then
+        wait
+        processes=0
+        echo "processes cleared at $processes"
+      fi
+
+      # Construct command
+      ./venv/bin/python3 aug_translation.py -i ${input} -o ${output} -j -l ${lang}&
```
Comment (klei22) [inline data_augmentation/tinystories_en.sh:37]: Change to `python3`

File: data_augmentation/tinystories_en.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  # Make sure there is an output directory
+  output_dir="./datasets/json_stories/${lang}"
+  if [ ! -d "${output_dir}" ]; then
+    mkdir -p "${output_dir}"
+  fi
+
+  # Loop datasets
+  for i in {00..49}
+  do
+    input="./datasets/json_stories/data${i}.json"
```
Comment (klei22) [inline data_augmentation/tinystories_en.sh:24]: Add 'archive' after json_stories

**Review by klei22 — CHANGES_REQUESTED** [review]
After trying, there are two changes needed.

File: data_augmentation/tinystories_en.sh (hunk 1 above)
Comment (gkielian) [reply data_augmentation/tinystories_en.sh]: Good catch! I changed this back to python3 (was switching between pip vs conda setups and forgot to remove the hotfix).

Comment (gkielian) [reply data_augmentation/tinystories_en.sh]: Sharing direct link to diff showing resolution: https://github.com/chipGPT/nanoGPT/pull/16/commits/d111aa50710799c3f20455ecc5c6b6e4e80400f8

**Review by klei22 — APPROVED** [review]

**Comment by klei22** [comment]
Changes look good.

**Pull request merged**